ios/Classes/BetterPlayer.h

```diff
@@ -62,6 +65,10 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)setIsLiveStream:(BOOL) isLiveStream;
 - (void)setPipSeekButtonsHidden:(BOOL) isHidden;
 - (void)setIsDisplayPipButtons:(BOOL) isDisplay;
+- (void)setIsPremiumBannerDisplay:(BOOL) isDisplay;
+- (void)setIsPreparingDatasource:(bool)isPreparingDatasource;
```

Comment:
```suggestion
- (void)setIsPreparingDatasource:(BOOL) isPreparingDatasource;
```
Please unify it with line above.